CONTRIBUTING.md

```diff
@@ -167,6 +167,20 @@ sbt
 akka-cluster/scalafmtAll
 akka-persistence/scalafmtAll
 ```
+And you can format the Scala source code with keeping the *import*s sorted with:
+
+```
+sbt
+project akka-actor
+sortImports
+```
+
+To format the Java source code:
```

Comment:
shouldn't this example be moved up, after the "To format the Scala source code"?